awsutil/awsutil_test.go

```diff
@@ -0,0 +1,76 @@
+package awsutil
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/stretchr/testify/require"
+)
+
+func TestECSTaskMeta(t *testing.T) {
+	taskRegion := "bogus-east-1"
+	nonTaskRegion := "some-other-region"
+	ecsMeta := ECSTaskMeta{
+		Cluster: "test",
+		TaskARN: fmt.Sprintf("arn:aws:ecs:%s:123456789:task/test/abcdef", taskRegion),
+		Family:  "task",
+	}
+
+	cases := []struct {
```

Comment:
Does't matter but worth noting another patter you'll see is `map[string]struct` and putting the test name as the key.